packages/vitest/src/index.ts

```diff
@@ -39,6 +39,10 @@ type Promisify<O> = {
 }
 
 declare global {
+  // eslint-disable-next-line @typescript-eslint/prefer-ts-expect-error
+  // @ts-ignore build global namespace conflict
+  let __vitest_worker__: WorkerGlobalState
```

Comment:
In order to export `VitestRunner` I had to make `__vitest_worker__` global.
Quite controversial 👀

Reply:
Maybe let's not type it, but with a util function to get?

Reply:
> Maybe let's not type it, but with a util function to get?

`getVitestWorker`?

Reply:
Yeah, something like that.

Reply:
And I still wouldn't be able to export `VitestRunner` because it uses `__vitest_worker__` (typescript gives errors on build)

Reply:
We should rewrite accessing global state like with `rpc`?

Reply:
I am fine with exposing to global, but secretly, we do @ts-expect-error on set/read and then reuse the function to be explicitly and type-safe.

> Should we rewrite accessing global state everywhere like with 

Certainly, can do in another PR.

Reply:
Let's do it then, will wait for a pr